### DISCUSSION TITLE
Remix build error on Vercel

### DISCUSSION THREAD

**Discussion by rafael-franca** [vercel/community#3505] [Help] [answered]
### Summary

I'm developing an application using Remix, but since 2 days ago build it's successfully however when try view application an error occurred. I'm tried to solve, but the error only occurs in Vercel. When try deploy on CloudFlare Pages, it's fine.

The error message is: `undefined is not an object (evaluating 'e[r.id].handle')`

### Example

https://vantagio-cwgzsyth4-rafaelfranca.vercel.app/

### Steps to Reproduce

In local build, it's fine. But in Vercel, this error occurs.

<img width="789" alt="image" src="https://github.com/vercel/community/assets/577574/62c0ba3b-3850-4391-82c5-7acb99c03fb6">

**Comment by amyegan** [comment]
Hey @rafael-franca. I wasn't able to replicate the issue with a [Remix boilerplate](https://vercel.com/templates/remix/remix-boilerplate) starter. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what is going wrong.

**Reply by rafael-franca** [reply, marked as answer]
@amyegan I discovered how the bug occur. I'm using the **Route File Naming V2**, and I have created a folder named **404**, and a file `route.tsx`, it's a normal process. However, I decided to remove this folder. So, at this moment, the build started a buggy. I'm tried with a clean project, and I can reproduce. I can't tried with a **Route File Naming V1**.
